libs/ngxtension/computed-from/src/computed-from.ts

```diff
@@ -59,53 +86,82 @@ export function computedFrom<Input extends object, Output = Input>(
  * }
  * ```
  */
-export function computedFrom(
-	sources: any,
-	operator?: OperatorFunction<any, any>,
-	injector?: Injector
-): Signal<any> {
+export function computedFrom<Input = any, Output = Input>(
+	...args: any[]
+): Signal<Output> {
+	const { normalizedSources, hasInitValue, operator, options } = _normalizeArgs<
+		Input,
+		Output
+	>(args);
+
+	let injector = options?.injector;
 	injector = assertInjector(computedFrom, injector);
+	/* try { //CUSTOM ERROR HANDLING FOR computedFrom */
+	//IF YOU PASS options.initialValue RETURN Signal<Output> WITHOUT ANY PROBLEM EVEN IF sources Observable ARE ASYNC (LATE EMIT) -> OUTPUT SIGNAL START EMITING SYNC initialValue!
+	//IF YOU DON'T PASS THE initialValue ENFORCE THAT Observable SYNC EMIT USING THE NATIVE toSignal requireSync:true OPTION -> SO IF ANYONE FORGET TO USE startWith IT WILL ERROR!
+	const ret: Signal<Output> = hasInitValue
+		? toSignal(combineLatest(normalizedSources).pipe(operator), {
+				initialValue: options?.initialValue!, //I'M SURE initialValue EXIST BECAUSE hasInitValue IS TRUE
+				injector: options?.injector, //EVENTUALLY PASSING toSignal THE injector TO USE THE CORRECT INJECTION CONTEXT
+		  })
+		: toSignal(combineLatest(normalizedSources).pipe(operator), {
+				injector: options?.injector, //EVENTUALLY PASSING toSignal THE injector TO USE THE CORRECT INJECTION CONTEXT
+				requireSync: true, //THIS WILL USE NATIVE toSignal BEHAVIOR THAT CHECK IF ALL OBSERVABLE EMIT SYNC OTHERWISE THROW ERROR
+				// -> SO IF ANYONE FORGET TO USE startWith IT WILL ERROR! THIS IS PREFERRED TO OLD "SPURIOUS SYNC EMIT" OF null OR Input ([], {})
+				//THAT CAN CAUSE RUNTIMES ERRORS THAT TS CAN'T CATCH BECOUSE THE OLD SIGNATURE Signal<Output> IS NOT "STRICTIER" FOR THOSE CASES!
+		  });
+	return ret;
+	/* //WE CAN DECIDE TO CUSTOMIZE THE ERROR TO BE MORE SPECIFIC FOR computedFrom
+	} catch (e: any) {
+		if ( //EURISTIC TO CHECK IF THE ERROR IS CAUSED BY requireSync
+			e.message.includes('requireSync') ||
+			e.message.includes('NG601') ||
+			e.code == 601
+		)
+			console.warn(
```

Comment:
@nartc What do you think about this one? 

I think we will need sth like this.

Reply:
@eneajaho when you ask me to open a new issue to discuss try catch handle error are you referring to those commented lines where I suggest a way to handle the error due to `requireSync` to customize the message for `computedForm`?